tomviz/DataPropertiesPanel.cxx

```diff
@@ -353,4 +361,30 @@ void DataPropertiesPanel::updateZLength()
   this->Internals->updateSpacing(2, newLength);
   this->updateData();
 }
+
+void DataPropertiesPanel::updateAxesGridLabels()
+{
+  vtkSMViewProxy *view = ActiveObjects::instance().activeView();
+  if (!view) {
+    return;
+  }
+  vtkSMProxy *axesGrid = vtkSMPropertyHelper(view, "AxesGrid").GetAsProxy();
+  DataSource *ds = ActiveObjects::instance().activeDataSource();
+  if (!axesGrid || !ds) {
+    return;
+  }
+  QString xTitle = QString("X Axis (%1)").arg(ds->getUnits(0));
```

Comment:
I think we could just label these X, Y, and Z - the axis label feels a little redundant to me.